### PR TITLE
Use '.jenkins-card' for cards

### PR DESCRIPTION
**Pull request by janfaracik**
See https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/496

Introduced in https://github.com/jenkinsci/jenkins/pull/9148 is the [card component](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/layout/card.jelly). This PR updates the graph view plugin to use those new styles, bringing it inline with core and removing a bunch of CSS. Also updated a few variables to Jenkins colours when possible.

#### Before

image

image

#### After

image

image

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/frontend/pipeline-graph-view/_cards.scss
Comment (timja) [inline src/main/frontend/pipeline-graph-view/_cards.scss:27]: likely this change causes graph overflow:

image

```
pipeline {
    agent any
    stages {
        stage('Non-Parallel Stage') {
            steps {
                echo 'This stage will be executed first.'
            }
        }
        stage('Parallel Stage') {
            failFast true
            parallel {
                stage('Branch A') {
                    steps {
                        echo "On Branch A"
                    }
                }
                stage('Branch B') {
                    steps {
                        echo "On Branch B"
                    }
                }
                stage('Branch C') {
                    stages {
                        stage('Nested 1') {
                            steps {
                                echo "In stage Nested 1 within Branch C"
                            }
                        }
                        stage('Nested 2') {
                            steps {
                                echo "In stage Nested 2 within Branch C"
                            }
                        }
                    }
                }
            }
        }
        stage('Skipped stage') {
            when {
                branch 'master'
            }
            steps {
                echo 'This stage will be skipped.'
            }
        }
        stage('Parallel Stage 2') {
            failFast true
            parallel {
                stage('Branch A') {
                    steps {
                        echo "On Branch A"
                    }
                }
                stage('Branch B') {
                    steps {
                        echo "On Branch B"
                    }
                }
                stage('Branch C') {
                    stages {
                        stage('Nested 1B') {
                            steps {
                                echo "In stage Nested 1B within Branch C"
                            }
                        }
                        stage('Nested 2B') {
                            steps {
                                echo "In stage Nested 2B within Branch C"
                            }
                        }
                    }
                }
            }
        }
    }
}
```

**Review by timja — CHANGES_REQUESTED** [review]
Looks really good only issue I found was overflow.

File: src/main/frontend/pipeline-graph-view/_cards.scss
Comment (janfaracik) [reply src/main/frontend/pipeline-graph-view/_cards.scss:27]: > likely this change causes graph overflow:
> 
> image
> ```
> pipeline {
>     agent any
>     stages {
>         stage('Non-Parallel Stage') {
>             steps {
>                 echo 'This stage will be executed first.'
>             }
>         }
>         stage('Parallel Stage') {
>             failFast true
>             parallel {
>                 stage('Branch A') {
>                     steps {
>                         echo "On Branch A"
>                     }
>                 }
>                 stage('Branch B') {
>                     steps {
>                         echo "On Branch B"
>                     }
>                 }
>                 stage('Branch C') {
>                     stages {
>                         stage('Nested 1') {
>                             steps {
>                                 echo "In stage Nested 1 within Branch C"
>                             }
>                         }
>                         stage('Nested 2') {
>                             steps {
>                                 echo "In stage Nested 2 within Branch C"
>                             }
>                         }
>                     }
>                 }
>             }
>         }
>         stage('Skipped stage') {
>             when {
>                 branch 'master'
>             }
>             steps {
>                 echo 'This stage will be skipped.'
>             }
>         }
>         stage('Parallel Stage 2') {
>             failFast true
>             parallel {
>                 stage('Branch A') {
>                     steps {
>                         echo "On Branch A"
>                     }
>                 }
>                 stage('Branch B') {
>                     steps {
>                         echo "On Branch B"
>                     }
>                 }
>                 stage('Branch C') {
>                     stages {
>                         stage('Nested 1B') {
>                             steps {
>                                 echo "In stage Nested 1B within Branch C"
>                             }
>                         }
>                         stage('Nested 2B') {
>                             steps {
>                                 echo "In stage Nested 2B within Branch C"
>                             }
>                         }
>                     }
>                 }
>             }
>         }
>     }
> }
> ```

Good spot, should be sorted now

**Review by timja — APPROVED** [review]
LGTM, there's a couple of pending changes I'll see if we can get in before we bump the baseline, otherwise we can merge this in a week or two if those don't get sorted

**Comment by janfaracik** [comment]
> LGTM, there's a couple of pending changes I'll see if we can get in before we bump the baseline, otherwise we can merge this in a week or two if those don't get sorted

Awesome, thanks!

**Pull request merged**
